### PR TITLE
Reservations

### PR DESCRIPTION
**Pull request by mmghannam**

### CODE REVIEW

File: controllers/sessions.js [hunk 1]
```diff
@@ -13,17 +14,21 @@ var index = function (req, res) {
...
+            model: User
         }]
     }).then(function (sessions) {
         res.status(200).send(sessions).end();
     }).catch(function (err) {
         res.status(400).send({ error: err }).end();
     });
-};
+}
+    ;
```
Comment (mmghannam) [inline controllers/sessions.js:31]: This doesn't look right, recheck for unmatched brackets

File: controllers/users.js [hunk 2]
```diff
@@ -167,12 +172,83 @@ function login(req, res) {
     });
 }
 
+// POST /user/:id/add/session/:sid
```
Comment (mmghannam) [inline controllers/users.js]: this needs to be /users/:id/add/session/:sid

File: controllers/sessions.js (hunk 1 above)
Comment (amrufathy) [reply controllers/sessions.js:31]: Nah, it's alright. The semi-colon is just misplaced.

File: controllers/users.js (hunk 2 above)
Comment (amrufathy) [reply controllers/users.js]: I didn't change it :smile:. Fix it.

**Pull request merged**